### PR TITLE
REPORT-804: I introduced a query very similar to DrugOrderFilter, whi…

### PR DESCRIPTION
**Pull request by IamMujuziMoses**
…ch supports the parameters provided

https://issues.openmrs.org/browse/REPORT-804

### CODE REVIEW

File: api/src/main/java/DrugOrderFilter.java [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+	private boolean onlyCurrentlyActive;
+	
+	
+	public DrugOrderFilter() {
+		
+		super.setType("Patient Filter");
+		super.setSubType("Drug Order Filter");
+	}
+	
+	@Override
```
Comment (sherrif10) [inline api/src/main/java/DrugOrderFilter.java:65]: @IamMujuziMoses  are you available for this, Can you include unit tests for these methods

**Comment by gracepotma** [comment]
@dkayiwa to review

**Comment by dkayiwa** [comment]
@IamMujuziMoses are you still working on this?

**Comment by IamMujuziMoses** [comment]
> 
> 
> @IamMujuziMoses are you still working on this?

@dkayiwa no thank you sir

**Pull request closed without merging**